main_test.go

```diff
@@ -176,8 +175,8 @@ func testCommands(t *testing.T, c *cli.Context, commands []*cli.Command, prefix
 func TestInitContext(t *testing.T) {
 	t.Parallel()
 
-	ctx := context.Background()
-	cfg := config.NewNoWrites()
+	ctx := config.NewContextReadOnly()
+	cfg := config.NewReadOnly()
```

Comment:
The context is supposedly set to the provided config below in initContext in our main, so keeping a plain one is fine here.